one/src/main.rs

```diff
@@ -181,22 +223,25 @@ impl Daemon {
         debug!(?p2p_config, "using p2p config");
 
         // Construct a recon implementation.
-        let recon = Arc::new(std::sync::Mutex::new(recon::Recon::from_set([].into())));
+        let recon = Arc::new(Mutex::new(Recon::from_set([].into())));
```

Comment:
This implies recon should have a default to call.

Reply:
Yes creating a new empty Recon would be a good function to add.